plugin-server/src/worker/vm/upgrades/export-events.ts

```diff
@@ -12,7 +12,7 @@ const EXPORT_BUFFER_BYTES_DEFAULT = 1024 * 1024
 const EXPORT_BUFFER_BYTES_MAXIMUM = 100 * 1024 * 1024
 const EXPORT_BUFFER_SECONDS_MINIMUM = 1
 const EXPORT_BUFFER_SECONDS_MAXIMUM = 600
-const EXPORT_BUFFER_SECONDS_DEFAULT = isTestEnv() ? EXPORT_BUFFER_SECONDS_MAXIMUM : 10
+const EXPORT_BUFFER_SECONDS_DEFAULT = isTestEnv() ? 0 : 10
```

Comment:
Note that for testing we set buffer seconds to 0, and leave as is for non-testing